esp-cargo

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
weird request but can be call this `.sh` so it's explicit?

Reply:
Since this script is meant to be used as an executable, I'd prefer to stick to the convention which is to have no extension:
https://google.github.io/styleguide/shellguide.html#s2.1-file-extensions

The [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) will run the script with bash without the user needing to specify.

I may change it to `.sh` if I later decide to add a Windows (`.bat`) version of the script to differentiate the two, but for now I only plan on supporting WSL/Linux and MacOS.